lib/routes/mangadex/user/follows.ts

```diff
@@ -0,0 +1,156 @@
+            source: ['mangadex.org/titles/follows', 'mangadex.org/titles/follows?tab=reading'],
+            target: '/user/follow/reading',
+        },
+    ],
+    features: {
+        requireConfig: [
+            {
+                name: 'MANGADEX_USERNAME',
+                description: 'MangaDex Username, required when refresh-token is not set',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_PASSWORD',
+                description: 'MangaDex Password, required when refresh-token is not set',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_CLIENT_ID',
+                description: 'MangaDex Client ID',
+                optional: false,
+            },
+            {
+                name: 'MANGADEX_CLIENT_SECRET',
+                description: 'MangaDex Client Secret',
+                optional: false,
+            },
+            {
+                name: 'MANGADEX_REFRESH_TOKEN',
+                description: 'MangaDex Refresh Token, required when username and password are not set',
+                optional: true,
+            },
+            {
+                name: 'MANGADEX_FILTERED_LANGUAGES',
+                description: 'MangaDex Filtered Languages',
+                optional: true,
+            },
+        ],
+    },
+    handler,
+};
+
+async function handler(ctx) {
+    const userFollowUrl = 'https://api.mangadex.org/manga/status';
+
+    const { type } = ctx.req.param();
+
+    const followType = (type || 'reading') as FollowType;
+
+    const accessToken = await getToken();
+
+    const statuses = (await cache.tryGet(
+        `mangadex:user-follow-${followType}`,
+        async () => {
+            const response = await got.get(userFollowUrl, {
+                headers: {
+                    Authorization: `Bearer ${accessToken}`,
+                    'User-Agent': config.trueUA,
+                },
+            });
+
+            const statuses = response?.data?.statuses;
+            if (!statuses) {
+                throw new Error('Failed to retrieve user follows from MangaDex API.');
+            }
+
+            return statuses;
+        },
+        config.cache.contentExpire
+    )) as Record<string, string>;
+
+    const mangaIds = filterByValue(statuses, statusMap[followType]);
+
+    const mangaMetaMap = await getMangaMetaByIds(mangaIds);
+
+    const mangaChapters = await Promise.all(mangaIds.map((id) => getMangaChapters(id, undefined, 10)));
+
+    const mangas = mangaChapters
+        .flatMap((chapters, index) => {
+            const mangaMeta = mangaMetaMap.get(mangaIds[index]);
+            return chapters.map((chapter) => ({
+                title: mangaMeta?.title ?? 'Unknown',
+                link: chapter.link,
+                pubDate: chapter.pubDate,
+                description: chapter.title ?? '',
+                image: mangaMeta?.cover ?? '',
+            }));
+        })
+        .flat();
```

Comment:
```suggestion
        });
```
No more flat required